fleet_vehicle_inspection/static/src/scss/backend.scss

```diff
@@ -1,62 +0,0 @@
-.o_inspection_image_list .o_kanban_view.o_kanban_ungrouped {
-    width: auto;
-
-    .o_kanban_record {
-        flex: 0 1 50%;
-        position: relative;
-
-        @include media-breakpoint-up(md) {
-            flex: 0 0 percentage(1/3);
-        }
-
-        @include media-breakpoint-up(lg) {
-            flex: 0 0 percentage(1/5);
-        }
-
-        @include media-breakpoint-up(xl) {
-            flex: 0 0 percentage(1/6);
-        }
-        // make the image square and in the center
-        .o_squared_image {
-            position: relative;
-            overflow: hidden;
-            padding-bottom: 100%;
-            > img {
-                position: absolute;
-                margin: auto;
-                top: 0;
-                left: 0;
-                bottom: 0;
-                right: 0;
-            }
-        }
-
-        .o_inspection_image_size {
-            position: absolute;
-            top: 0;
-            left: 0;
-        }
-    }
-}
-
-.o_inspection_image_modal {
-    .o_inspection_image_modal_container {
-        border-left: 1px solid gray("400");
-
-        .o_field_image {
-            margin-bottom: 0;
-            box-shadow: 0 2px 10px gray("300");
-
-            > img {
-                border: 1px solid gray("400");
-                height: 200px;
-                width: auto;
-            }
-
-            .o_form_image_controls {
-                @include o-hover-opacity(0.7, 1);
-                padding: 2%;
-            }
-        }
-    }
-}
```

Comment:
Has the removal of these styles been checked?

I didn't like having these, to be honest, so I'm happy if we can remove it.. but are you sure there's no bad side effects?

Could you compare the screenshots of the view in v14 and now?

Reply:
Yes, it has been checked and there are no side effects. Here are the prints of the two screens:

Version 15 - local
![image](https://user-images.githubusercontent.com/56009318/231580033-3755f81c-8a28-4ccc-9fb3-13c3f7a698a9.png)

Version 14 - runboat
![image](https://user-images.githubusercontent.com/56009318/231580235-9e397a2b-9ddd-4f48-9730-f107ccf3ca5b.png)